website/src/pages/contributing/contributors/index.tsx

```diff
@@ -33,8 +34,22 @@ const Index = () => {
       <p>Location: {contributor.location}</p>
       <p>{contributor.contributions} contributions to master</p>
     </div>
-  </div>
-    ,);
+  </div>,);
+
+  // Create NCC element
+  const ncc: React.JSX.Element[] = ncContributors.map((item,) => <div
+    className={'card profile'}
+    key={item.id}>
+    <img src={'#'} alt={item.name} />
+    <div>
+      <h2>
+        <ExternalLink to={item.url} label={item.name} />
+      </h2>
+      <p>{item.intro}</p>
+      <p>Location: {item.location}</p>
+      <p>{item.contributions} contributions to master</p>
```

Comment:
This won't ever be filled, since the none code contributors didn't contribute code - otherwise they'd be in the normal contributor's list already